.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: true
```

Comment:
Is this an problem?
You can open up Blank_Issues
![image](https://user-images.githubusercontent.com/25481501/99702917-ba0ff100-2a96-11eb-8ed6-919bd123504e.png)

I actually think disabling blank issues is an better idea, because their is no real use for it. The issues template provide most information for issues.

Reply:
Ok